src/Display/Modal/Modal.tsx

```diff
@@ -192,16 +195,26 @@ export const Modal: FC<Props> = ({
 export type sizeType = 's' | 'm' | 'l' | 'xl';
 
 export interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'title'> {
-  title?: ReactNode; // we don't really use this title attribute for div element
+  /** Sets title of Modal. */
+  title?: ReactNode;
+  /** Sets the visibility of Modal. */
   isVisible: boolean;
+  /** Sets to close Modal. */
   onClose?(): void;
-  hideContentArea?: boolean;
-  centering?: boolean;
-  removeAnimation?: boolean;
   /** An array containing 0-2 elements. */
   footer?: ReactElement[];
+  /** Centering Modal. */
+  centering?: boolean;
+  /** Removes animation when opening and closing the modal. */
+  removeAnimation?: boolean;
+  /** Sets size of Modal. */
   size?: sizeType;
+  /** Hides header area. */
   hideHeader?: boolean;
+  /** Hides content area. */
+  hideContentArea?: boolean;
+  /** Whether to unmount child components on onClose */
+  destroyOnClose?: boolean;
```

Comment:
I'm wondering if we should rather name this `destoryChildrenOnClose` because the rest of the modal does not really get unmounted on close.

Reply:
Yep, specify `children` in the name is better. And I just felt the default boolean value should be `false`, so I rename it to `keepChildrenMountedOnClose` which taking `material-ui`'s `keepMounted` as reference https://material-ui.com/api/modal/